google/ggrc-core
# BC: GGRC-7715 Add subtree for assessments on Bulk complete modal

# Comment

**Should be merged to "asmt-bulk".**

# Dependencies

This PR is `on hold` until the dependencies are merged:

- [x] #9989

# Issue description

 Add subtree for assessments on Bulk complete modal.

# Steps to test the changes

1. Open 'Bulk complete' modal from 'My Assessments' or 'Audit' page. In 'Select assessments to complete in bulk' section you should see three subtree’s section (Mapped objects, Evidence and Custom Attributes, Comments).
![Subtree](https://user-images.githubusercontent.com/25364800/64608505-7d952f80-d3d3-11e9-9504-8d608af2de6d.png)
2. 'Mapped objects' subtree section should display snapshots of mapped to Assessment objects.
![Mapped_objects](https://user-images.githubusercontent.com/25364800/64608512-81c14d00-d3d3-11e9-8c2c-b6ec69d55396.png)
3. 'Evidence and Custom Attributes' subtree section should display evidence files and urls attached to Assessment and LCA.
![Evidence LCA](https://user-images.githubusercontent.com/25364800/64608519-8554d400-d3d3-11e9-8f08-f02a1aa76005.png)
4. 'Comments' subtree section should display comments from related Assessment.
![Comments](https://user-images.githubusercontent.com/25364800/64608527-8980f180-d3d3-11e9-8a57-781589b35e6a.png)
If the total number of comments is more than 5 you should see message "Showing up to 5 related comments. See more". "See more" link should open Assessment in a new tab.
![More_comments](https://user-images.githubusercontent.com/25364800/64608169-bbde1f00-d3d2-11e9-8caf-30368d5b97d1.png)



# Solution description
Uses QueryAPI request to load data for subtree sections. Request is sent only when you click on the the appropriate subtree section. Loaded data is cached in order to not load the same information one more time.

# Sanity checklist




# PR Review checklist
